src/Security/CheckGroups/CheckGroupServerConfig.js

```diff
@@ -50,14 +51,25 @@ class CheckGroupServerConfig extends CheckGroup {
       }),
       new Check({
         title: 'Client class creation disabled',
-        warning: 'Attackers are allowed to create new classes without restriction and flood the database.',
-        solution: 'Change Parse Server configuration to \'allowClientClassCreation: false\'.',
+        warning:
+          'Attackers are allowed to create new classes without restriction and flood the database.',
+        solution: "Change Parse Server configuration to 'allowClientClassCreation: false'.",
         check: () => {
           if (config.allowClientClassCreation || config.allowClientClassCreation == null) {
             throw 1;
           }
         },
       }),
+      new Check({
+        title: 'Users are created without public access',
+        warning: 'Users are created with public read access.',
```

Comment:
The warning should describe why this is a security risk. Could you add a short explanation?
For example (if that is an accurate description):
```
warning: 'Users with public read access are exposed to anyone who knows their object IDs.'
```